lib/lookup.js

```diff
@@ -76,20 +76,25 @@ function resolve(context, next) {
       if (rep.stripAnsi) {
         context.module.stripAnsi = rep.stripAnsi;
       }
-      if (rep.replace) {
-        if (!rep.repo && !meta.repository) {
-          next(new Error('no-repository-field in package.json'));
-          return;
+      if (!rep.npm) {
+        if (typeof rep.replace === 'boolean' && rep.replace === false) {
+          rep.npm=true;
         }
-        context.emit('info','lookup-found',detail.name);
-        var url = makeUrl(
+        if (!rep.npm){
```

Comment:
why the second equality check here, see above comment regarding hoisting the replace check to avoid having to check twice